### PR TITLE
debug,pr,pj return submitted var

### PR DESCRIPTION
**Pull request by ionas**
Ref: https://github.com/cakephp/cakephp/pull/8534#issuecomment-201290080

### CODE REVIEW

File: en/core-libraries/global-constants-and-functions.rst [hunk 1]
```diff
@@ -97,6 +97,10 @@ such as debugging and translating content.
 
 .. php:function:: debug(mixed $var, boolean $showHtml = null, $showFrom = true)
 
+    .. versionadded:: 3.2.6
```
Comment (markstory) [inline en/core-libraries/global-constants-and-functions.rst]: Needs to be 3.3.0 instead.

Comment (antograssiot) [inline en/core-libraries/global-constants-and-functions.rst]: so the PR should target 3.3

Comment (markstory) [inline en/core-libraries/global-constants-and-functions.rst]: Yes.

**Comment by ionas** [comment]
Can you change the target from 3.0 to 3.3?

**Pull request closed without merging**